### ISSUE TITLE
EBImage failing in 3.20

### ISSUE THREAD

**Issue by almahmoud** [Bioconductor/bioconductor_docker#112]
EBImage is failing in 3.20 with
```
Error in dyn.load(file, DLLpath = DLLpath, ...) :
  unable to load shared object '/usr/local/lib/R/site-library/tiff/libs/tiff.so':
  libtiff.so.5: cannot open shared object file: No such file or directory
```

Likely due to missing sysdep, but noting here to look into it later

**Comment by almahmoud** [comment]
`ln -s /usr/lib/x86_64-linux-gnu/libtiff.so.6 /usr/lib/x86_64-linux-gnu/libtiff.so.5` seems to bypass the issue. Uninstalling libtiff6-dev and only installing libtiff5-dev doesn't work, libtiff5-dev seems to always come with libtiff6. Haven't looked too much into it though

**Comment by vjcitn** [comment]
Within the 3.20 container, I got EBImage to install by installing the CRAN source package for tiff with R CMD INSTALL prior to  BiocManager::install("EBImage")
```
wget https://cran.r-project.org/src/contrib/tiff_0.1-12.tar.gz
R CMD INSTALL tiff_0.1-12.tar.gz
# now in R use BiocManager::install("EBImage")
```

**Comment by almahmoud** [comment]
Good catch... I think the issue is that it's trying to use a wrong binary

**Comment by grimbough** [comment]
I just ran into this issue and wondered if there was any update.  Can the `tiff` package binary be rebuilt to link against the correct version of libtiff?  I'm not sure I understand whose responsbility it is to build the binaries of CRAN packages that are distgrubuted by Bioconductor. Happy to report this elsewhere if appropriate.

**Comment by vjcitn** [comment]
Any need to keep this issue open?

**Comment by Bisaloo** [comment]
For our use case ([online version of the "Modern Statistics for Modern Biology"](https://www.huber.embl.de/msmb/)), it looks like this is fixed and we can safely use the tiff package version provided in the `bioconductor:latest` (currently 3.22) Docker container.

**Comment by almahmoud** [comment]
Thank you for the confirmation @Bisaloo
